setup.cfg

```diff
@@ -1,12 +1,3 @@
 # For details, see
 #   https://docs.python.org/2/distutils/configfile.html
 #   https://packaging.python.org/distributing/#manifest-in
-
-[nosetests]
```

Comment:
Shouldn't this part be replaced by pytests equivalent ?

Reply:
The configuration of `pytest` is done in the command